### PR TITLE
Release/16.0.5

### PR DESCRIPTION
**Pull request by qubis741**
### Added

- `Toggle`: `tooltip` and `tooltipPosition` props. ([@qubis741](https://github.com/qubis741)) in [#2355](https://github.com/teamleadercrm/ui/pull/2355))

### CODE REVIEW

**Review by eniskraasniqi1 — DISMISSED** [review]

**Review by eniskraasniqi1 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +18,18 @@
 
 - `Dialog`: Fix close button alignment in cases where css order changes during build. ([@lorgan3](https://github.com/lorgan3)) in [#2351](https://github.com/teamleadercrm/ui/pull/2351))
 
+## [16.0.5] - 2022-09-16
```
Comment (lorgan3) [inline CHANGELOG.md:21]: B: something is off here

image

**Review by lorgan3 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (qubis741) [reply CHANGELOG.md:21]: no, 16.0.4. wasn't marked in changelog, so I have added it

**Review by eniskraasniqi1 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +12,12 @@
 
 ### Dependency updates
 
+## [16.0.5] - 2022-09-16
+
+### Added
+
+- `Toggle`: `tooltip` and `tooltipPosition` props. ([@qubis741](https://github.com/qubis741)) in [#2355](https://github.com/teamleadercrm/ui/pull/2355))
```
Comment (lorgan3) [inline CHANGELOG.md:19]: Sorry should've already checked this in my last review 🙈 
This is a new feature, so should be `16.1.0` right?

Comment (qubis741) [reply CHANGELOG.md:19]: oh yeah, true, thanks

**Comment by qubis741** [comment]
Closed in favor of proper version -> https://github.com/teamleadercrm/ui/pull/2361

**Pull request closed without merging**